builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -644,6 +688,54 @@ local function sort_and_filter_pkgs()
 	store.filter_packages(search_string)
 end
 
+-- Resolves the package specification stored in auto_install_pkg into an actual package.
+local function prepare_auto_install_pkg()
```

Comment:
prepare_auto_install_pkg is only called in load, so if you load the menu once and then attempt to autoinstall it won't work. This will cause issues when using this feature in #13807

Could prepare_auto_install_pkg and do_auto_install_pkg be merged into a single instruction, where auto_install_pkg is only ever the key?

Reply:
Good catch. What do you think about this? a345824d7ed075caccab9a8e7826e5a36150a5d4